### PR TITLE
status page for ELB

### PR DESCRIPTION
**Pull request by elisehuard**

### CODE REVIEW

**Comment by acron0** [comment]
This is still inside the `api/` context, is that ok?

**Comment by elisehuard** [comment]
@acron0 I think it is since the nginx will be rerouting based on /api subpath?

**Comment by acron0** [comment]
Looking at [this file] (https://github.com/MastodonC/witan.ui/blob/feature/zip-script/start-nginx.sh) I think it'd work either way, but outside has its own access logging.

**Comment by sw1nn** [comment]
hmmm, I think as a sibling to api would be preferable, but if it takes more than 2 mins, we can live with it at /api/_elb_status

**Comment by sw1nn** [comment]
I don't have any strong rationale for the above, more of a gut feel, so happy to hear an alternative rationale

File: src/witan/app/handler.clj [hunk 1]
```diff
@@ -102,6 +102,13 @@
   (sweet/swagger-docs
    {:info {:title "Witan API"
            :description "Back-end API for the Witan project"}})
+  (sweet/context* "/monitoring" []
```
Comment (sw1nn) [inline src/witan/app/handler.clj]: sweet!

**Pull request merged**